test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -390,6 +390,48 @@ public async Task DotnetListPackage_OutdatedWithNoVersionsFound_Succeeds()
                 // Assert
                 var lines = listResult.AllOutput.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
                 Assert.True(lines.Any(l => l.Contains("packageX") && l.Contains("Not found at the sources")), "Line containing 'packageX' and 'Not found at the sources' not found: " + listResult.AllOutput);
+
+            }
+        }
+
+        [PlatformTheory(Platform.Windows)]
+        [InlineData("1.0.0", "", "2.1.0")]
+        public async Task DotnetListPackage_Outdated_Transitive_Succeeds(string currentVersion, string args, string expectedVersion)
```

Comment:
I think the test name can be improved. 
Maybe something like: 

```suggestion
        public async Task DotnetListPackage_Outdated_ProjectReferences_AreNotDisplayed(string currentVersion, string args, string expectedVersion)
```

Reply:
I have also created a tracking issue to add proper tests for dotnet list --deprecated.
https://github.com/NuGet/Home/issues/8481